doc/authorization-config.md

```diff
@@ -451,7 +489,7 @@ authorization: {
     rules: [
             {
                 match-request: {...}
-                allow: [ node1, node2, node3 ]
+                allow: [ node1, node2, node3, {extensions:{ext_shortname: value}} ]
```

Comment:
Might be good to show an example with more than one extension in it - either here or as an additional one.